web_sdk/web_test_utils/lib/skia_client.dart

```diff
@@ -108,6 +108,11 @@ class SkiaGoldClient {
     }
   }
 
+  Future<ProcessResult> _runCommand(List<String> command) {
+    print(command.join(' '));
```

Comment:
For testing you might need to abstract this logging function a little bit, like `printWarning` [here](https://github.com/flutter/engine/pull/24827).

Reply:
I merged this PR as we are still seeing some issues and need this extra logging ASAP. I'll work on the tests later.